crates/solvers-dto/src/solution.rs

```diff
@@ -84,30 +84,33 @@ pub struct JitOrder {
     pub signature: Vec<u8>,
 }
 
-#[derive(Debug, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub enum Kind {
     Sell,
     Buy,
 }
 
-#[derive(Debug, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 #[serde(tag = "kind", rename_all = "camelCase")]
 pub enum Interaction {
     Liquidity(LiquidityInteraction),
     Custom(CustomInteraction),
 }
 
-#[derive(Debug, Serialize)]
+#[serde_as]
+#[derive(Clone, Debug, Serialize)]
 #[serde(tag = "kind", rename_all = "camelCase")]
 pub struct Call {
     pub target: H160,
     pub value: U256,
+    #[serde(rename = "callData")]
+    #[serde_as(as = "serialize::Hex")]
```

Comment:
The `solvers-dto` fields don't match what the solver expects. We didn't notice that so far because we didn't have a test (or solver) using these new interactions. That means we also don't have to update the `solvers` repo before merging this.

Reply:
nit: should it be call_data be instead (would lead to the same naming and be less one-offy)?

Reply:
I agree. I just wanted to have the regular interactions and pre- and post-interactions consistent for this test and if we were to fix it for the new interactions this PR would likely introduce a bunch of renames that distract from the relevant changes. Can do that in a separate PR.

Reply:
Good catch!